tests/unit/scatter-test.js

```diff
@@ -63,3 +63,38 @@ test('Margins are the right size when showLegend is no', function(assert) {
   assert.equal(component.get('marginRight'), 30, 'right margin exists');
   assert.equal(component.get('marginBottom'), 0, 'no bottom margin if showLegend is no');
 });
+
+/**
+ * Sets up `scatterContent` to be used in the domaing padding test.
+**/
+function setupDomainPaddingTest() {
+  scatterContent.graphicLeft = 40;
+  scatterContent.graphicTop = 40;
+
+  scatterContent.numXTicks = 7;
+  scatterContent.graphicWidth = 630;
+  scatterContent.graphicHeight = 630;
+  scatterContent.graphPadding = 0.1;
+}
+
+test('Test domain with domain padding', function(assert) {
+  setupDomainPaddingTest();
+  scatterContent.hasXDomainPadding = true;
+  scatterContent.hasYDomainPadding = true;
+
+  var component = this.subject(scatterContent);
+
+  assert.equal(component.get('xScale')(0.017), 618.025);
```

Comment:
I would include a comment at least. If this test fails, it would be very hard for someone to understand why or how to fix it